conda_build/build.py

```diff
@@ -2093,7 +2118,13 @@ def bundle_conda(output, metadata: MetaData, env, stats, **kw):
     return final_outputs
 
 
-def bundle_wheel(output, metadata: MetaData, env, stats):
+def bundle_wheel(
+    output,
+    metadata: MetaData,
+    env,
+    stats,
+    new_prefix_files: set[str],
```

Comment:
Same comment here about the API breakage. Not sure how prominent the usage of wheel outputs is, though.

Reply:
https://github.com/conda/conda-build/pull/5216#discussion_r1534003328